### ISSUE TITLE
Inability to tab to accordion plus/minus 'control' flagged in accessibility audit

### ISSUE THREAD

**Issue by 36degrees** [alphagov/govuk-frontend#1770]
From a report on Slack:

> Hi, we recently had an accessibility review on a service and one of the issues identified was with the accordion component: Keyboard Only Users Cannot Expand Individual Accordions on ‘thank you for helping’ Page
> 2.1.1 Keyboard
> When navigating the ‘thank you for helping’ page, it has been observed that  users are unable to reach the ‘+’ icons to expand the accordion content.
> S1.10 Recommendation:
> Ensure that all user interface components are focusable and interactive using a keyboard only.
> 
> Could also be that the + on the right isn't selected by tabbing to it though it can by selected by mouse/pointer. This gives the impression that you can't do whatever the + lets you do - though it's the same as selecting the text to the left and hitting return

**Comment by 36degrees** [comment]
The plus/minus icon is not a separate control – the entire block is a control, and you can click anywhere between the heading and the plus and it will do the same thing. The focus state, which has changed since it was contributed, is intentionally smaller, because the entire control being yellow with a massive black underline was way too overpowering, especially as it sticks around when you’ve clicked the control.

I can see why they raised it, but at first glance at least I’m not sure we would look to change anything unless we saw it actually causing confusion and problems for users?

**Comment by nickcolley** [comment]
It's not clear from the feedback if the user was confused that the highlighted section was not related to the + icon and if they failed to proceed with the task.

We could consider highlighting the icon too but I worry that would be confusing too as there's two things focused?

<img width="679" alt="" src="https://user-images.githubusercontent.com/2445413/77562752-565a0780-6eb8-11ea-9b74-e9e78eeabd42.png">

```css
.js-enabled .govuk-accordion__section-button:focus + .govuk-accordion__icon {
    outline: 3px solid $govuk-focus-colour;
    background-color: $govuk-focus-colour;
    box-shadow: -3px 7px $govuk-focus-text-colour, 3px 7px $govuk-focus-text-colour;
}
```

With this in mind I would say we need more evidence that this preventing someone from understanding and successfully interacting with the component and close this for now.

**Comment by 36degrees** [comment]
I also think that's confusing, and may do more harm than good given we don't know it's actually an issue for users.

I'm leaning towards closing this too, but let's discuss it as a team at the next triage session.

**Comment by kellylee-gds** [comment]
Closing this for now but will monitor issue in case there are other reports of this being a problem for users.
